sdk/identity/identity/test/node/clientCertificateCredential.spec.ts

```diff
@@ -76,4 +78,39 @@ describe("ClientCertificateCredential", function() {
       );
     }
   });
+
+  it("sends a correctly formated token request with tracer", async () => {
+    const tracer = tracing.start({ samplingRate: 1 }).tracer;
+    TracerProxy.setTracer(tracer, SupportedPlugins.OPENCENSUS);
+    const tenantId = "tenantId";
+    const clientId = "clientId";
+    const mockHttpClient = new MockAuthHttpClient();
+
+    const credential = new ClientCertificateCredential(
+      tenantId,
+      clientId,
+      path.resolve(__dirname, "../test/azure-identity-test.crt"),
+      mockHttpClient.identityClientOptions
+    );
+
+    await tracer.startRootSpan({ name: "main" }, async (rootSpan) => {
+      await credential.getToken("scope", {
+        spanOptions: {
+          parent: rootSpan
+        }
+      });
+
+      assert.strictEqual(
+        rootSpan.numberOfChildren,
+        1,
+        `Number of children ${rootSpan.numberOfChildren} is not equal to 1.`
+      );
+      const numberOfDescendants = rootSpan.allDescendants().length;
```

Comment:
Should we add some kind of validator for the rootSpan that can understand more than just the child count / descendant count? 

In chromium we'd use golden files for this and write out all the events with things like timestamps sanitized, but maybe in this case we could just have a helper function that takes a tree-like structure of the expected events?